### PR TITLE
feat(alpha-test-app, OnyxSelect): Update alpha-test-app and OnyxSelect

### PR DESCRIPTION
**Pull request by MajaZarkova**
Relates to #568 

Added OnyxBadge example in the alpha-test-app and replaced the temporary Badge styles with OnyxBadge in the OnyxSelect component.

## Checklist

- [x] The added / edited code has been documented with [JSDoc](https://jsdoc.app/about-getting-started)
- [x] All changes are documented in the documentation app (folder `apps/docs`)

### CODE REVIEW

File: packages/sit-onyx/src/components/OnyxSelect/OnyxSelect.vue [hunk 1]
```diff
@@ -223,6 +223,10 @@ const { densityClass } = useDensity(props);
     }
   }
 
+  &__badge {
+    display: flex;
```
Comment (BoppLi) [inline packages/sit-onyx/src/components/OnyxSelect/OnyxSelect.vue:227]: I'd suggest block
```suggestion
    display: block;
    cursor: pointer;
```
here as we don't flex anything :)

File: packages/sit-onyx/src/components/OnyxSelect/OnyxSelect.vue [hunk 2]
```diff
@@ -120,9 +120,9 @@ const { densityClass } = useDensity(props);
         while the trigger arrow needs to point to the badge in the future.
         https://github.com/SchwarzIT/onyx/issues/763 -->
         <OnyxTooltip v-if="previewBadgeNumber" :text="selectionText" position="bottom">
-          <!-- TODO: use OnyxBadge component once it is implemented 
-          https://github.com/SchwarzIT/onyx/issues/565 -->
-          <div class="onyx-badge">{{ previewBadgeNumber }}</div>
+          <OnyxBadge class="onyx-select__badge" variation="neutral">{{
```
[the remark this answers is not in the document]
Comment (BoppLi) [reply packages/sit-onyx/src/components/OnyxSelect/OnyxSelect.vue:123]: I added it as suggestion :)

**Review by BoppLi — CHANGES_REQUESTED** [review]

File: packages/sit-onyx/src/components/OnyxSelect/OnyxSelect.vue (hunk 2 above)
[the remark this answers is not in the document]
Comment (BoppLi) [reply packages/sit-onyx/src/components/OnyxSelect/OnyxSelect.vue:123]: I added it as suggestion below :)

[the remark this answers is not in the document]
Comment (MajaZarkova) [reply packages/sit-onyx/src/components/OnyxSelect/OnyxSelect.vue:123]: Sure :)

**Review by BoppLi — APPROVED** [review]

**Pull request merged**